### PR TITLE
Fix issue #103

### PR DESCRIPTION
**Pull request by DanielVandH**
The `on 0..1` was added to make it consistent with printing in ClassicalOrthogonalPolynomials

### CODE REVIEW

**Comment by TSGut** [comment]
Don't forget to update the example in the readme. 😇 (tbf that readme is incredibly stale and needs a lot of fixing)

**Comment by DanielVandH** [comment]
I think there's nothing to update in the README anyway since I didn't add `0..1` into the printing of `SemiclassicalJacobi` (if we want consistency with ClassicalOrthogonalPolynomials, I think it would instead be `SemiclassicalJacobi(t, a, b, c)` instead of `SemiclassicalJacobi with weight...` anyway).

**Comment by TSGut** [comment]
I think we do want consistency wherever it isn't artificial. I think it makes sense to keep the weight explicitly mentioned however because semiclassicaljacobi isn't as much of a conventional term and there are other definitions floating around.

**Comment by DanielVandH** [comment]
That makes sense. I've added the interval into the printing now

File: README.md [hunk 1]
```diff
@@ -11,8 +11,9 @@ This builds on top of [ClassicalOrthogonalPolynomials.jl](https://github.com/Jul
 
 For example, the following gives a half-range Chebyshev polynomial:
 ```julia
+julia> usnig SemiclassicalOrthogonalPolynomials
```
Comment (TSGut) [inline README.md:14]: using

Comment (DanielVandH) [reply README.md]: That's what I get for thinking I can skip copy-pasting

**Comment by TSGut** [comment]
Happy for me to merge this? Maybe up the version number as well

**Comment by DanielVandH** [comment]
Good to go now, I've bumped the version

**Pull request merged**